tests/testthat/test-selections.R

```diff
@@ -310,6 +310,9 @@ test_that("old recipes from 1.0.1 work with new get_types", {
   )
   expect_false(identical(old_rec_sac, rec_sac))
 
+  # Avoid issue with new ptype field in 1.1.0
```

Comment:
```suggestion
  # Avoid issue with new ptype field in 1.1.0 (#1329)
```

Reply:
Am I correct in reading that this PR means that our backward compatibility for recipes generated before 1.0.1 is now broken? Or is it just that the results aren't identical even if an error is not thrown?

Reply:
it just means that you can't do `identical(rec_pre_1.1.0, rec_post_1.1.0)` because we added a field.

This "breaks" each time we are adding field in recipe objects, or new arguments in steps. So this is a minor thing, and it is tested.

Reply:
Awesome, sounds good.